temoa/model_checking/unit_checking/relations_checker.py

```
@@ -0,0 +1,476 @@
+        line_error_msg = f'Non-processed rows (see earlier tests): {listed_lines}'
+        error_msgs.append(line_error_msg)
+
+    return res, error_msgs
+
+
+def _column_exists(conn: sqlite3.Connection, table: str, column: str) -> bool:
+    """Check if a column exists in a table."""
+    try:
+        cursor = conn.execute(f'PRAGMA table_info({table})')
+        columns = [row[1] for row in cursor.fetchall()]
+        return column in columns
```

Ruff (0.14.7)

Consider moving this statement to an `else` block

(TRY300)